app/javascript/packs/thankYou/customJavascripts/closeButton.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const closeButton = document.getElementsByClassName('close')[0];
```

Comment:
if you add listener to one element use # id

Reply:
changed to document.getElementById('close')